### PR TITLE
Update register-devices-self.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about editing the file in excel and saving it as CSV make the file unusable

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5505

File: microsoft-365/managed-desktop/get-started/register-devices-self.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ You'll need to have the data in the CSV files combined into a single file to com
 
 `Import-CSV -Path (Get-ChildItem -Filter *.csv) | ConvertTo-Csv -NoTypeInformation | % {$_.Replace('"', '')} | Out-File .\aggregatedDevices.csv`
 
+> [!NOTE]
+> Extra columns are not supported. Quotes are not supported. Only ANSI-format text files can be used (not Unicode). Headers are case-sensitive. Editing the file in Excel and saving as a CSV file will not generate a usable file due to these requirements. Please make sure that any leading zeroes are preserved in the device serial numbers.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/register-devices-self.md]:
```suggestion
> Extra columns are not supported. Quotes are not supported. Only ANSI-format text files can be used (not Unicode). Headers are case-sensitive. Editing the file in Excel and saving it as a CSV file will not generate a usable file due to these requirements. Please make sure that any leading zeroes are preserved in the device serial numbers.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

**Comment by jaimeo** [comment]
Copy editors: we do not use "please."

@jwmurraycmu are you OK with the tech accuracy of this edit?

**Comment by jaimeo** [comment]
@AlexBlain are you OK with the tech accuracy of this edit?

**Comment by AlexBlain** [comment]
@jaimeo fine with that

**Pull request merged**